res/res_pjsip_session.c

```diff
@@ -4053,15 +4053,21 @@ static int new_invite(struct new_invite *invite)
 	 * so let's go ahead and send a 100 Trying out to stop any
 	 * retransmissions.
 	 */
+	if (pjsip_inv_initial_answer(invite->session->inv_session, invite->rdata, 100, NULL, NULL, &tdata) != PJ_SUCCESS) {
+		if (tdata == NULL) {
```

Comment:
This needs to be updated according to [coding guidelines](https://docs.asterisk.org/Development/Policies-and-Procedures/Historical-Policies-and-Procedures/Code-Review/Coding-Guidelines/).

if (!tdata) {
} else {
}

Reply:
if (!tdata) {